casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -84,7 +84,7 @@ object BlockCreator {
         now              <- Time[F].currentMillis
         invalidBlocksSet <- dag.invalidBlocks
         invalidBlocks    = invalidBlocksSet.map(block => (block.blockHash, block.sender)).toMap
-        unsignedBlock <- if (deploys.nonEmpty || parents.length > 1) {
+        unsignedBlock <- if (deploys.nonEmpty) {
```

Comment:
This is a bugfix so it would make sense to cover it with a test to make sure it doesn't get accidentally reverted

Reply:
PTAL